xray/default_emitter_test.go

```diff
@@ -89,35 +89,35 @@ func TestStreamingSegmentsOnGrandchildNode(t *testing.T) {
 	assert.Equal(t, 23, len(packSegments(root, nil)))
 }
 
-func TestStreamingSegmentsTreeHasOnlyOneBranch(t *testing.T) {
```

Comment:
Why did we comment this out ?

Reply:
same reason as https://github.com/aws/aws-xray-sdk-go/pull/177#discussion_r368186749